tools/data_processing/prep_funcs.py

```diff
@@ -0,0 +1,46 @@
+import datasets
+
+def prepare_narrativeqa(dataset: datasets.Dataset):
```

Comment:
move the prepare function in musique and qmsum here

Reply:
Sure.